CONTRIBUTING.md

```diff
@@ -143,6 +143,13 @@ package on the public dev feed, e.g. [@azure/mcp@0.0.10-beta.4799791](https://de
 
 Only manual runs of the pipeline sign and publish packages.  Building `main` or `hotfix/*` will publish to `npmjs.com`, all other refs will publish to the [public dev feed](https://dev.azure.com/azure-sdk/public/_artifacts/feed/azure-sdk-for-js).
 
+Packages published to the npmjs.com will always use the `@latest` [dist-tag](https://docs.npmjs.com/downloading-and-installing-packages-locally#installing-a-package-with-dist-tags).
+
+Packages published to the dev feed will use:
+- `@latest` for the latest official/release build
```

Comment:
I don't see anywhere below you set latest tag explicitly.

Reply:
Release.yml (official builds) uses `Tag: latest` and `ProductState: latest`
Integration.yml (ci and manual-not-main builds) uses `Tag: dev` and `Tag: pre`